src/v/kafka/server/connection_context.cc

```diff
@@ -72,6 +72,14 @@ ss::future<> connection_context::process_one_request() {
                       return ss::make_ready_future<>();
                   }
                   return dispatch_method_once(std::move(h.value()), s)
+                  .handle_exception_type(
+                    [this](const kafka_api_version_not_supported_exception& e) {
+                        vlog(
+                          klog.warn,
+                          "Unsupported API version in request from {} - {}",
+                          _rs.conn->addr,
+                          e.what());
```

Comment:
I think e.what() will print the same exact message

Reply:
almost, without an address

Reply:
Yean so maybe change to `vlog(klog.warn, {} - {}, e.what(), _rs.conn->addr);`